examples/app.py

```diff
@@ -23,34 +23,166 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 
-"""Minimal Flask application example for development.
+r"""Minimal Flask application example for development.
 
-Run example development server:
+Usage:
 
-.. code-block:: console
+1. Install elasticsearch:
```

Comment:
Do we need this part about installing elasticsearch? It's debian friendly only and i think it would be more useful to just link to elasticsearch manual instead.